hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DistributedFileSystem.java

```diff
@@ -380,6 +380,8 @@ public FSDataInputStream open(PathHandle fd, int bufferSize)
 
   @Override
   public String getErasureCodingPolicyName(FileStatus fileStatus) {
+    if (!(fileStatus instanceof HdfsFileStatus))
```

Comment:
has to have curly braces round. Not for us, but for our successors in maintaining this code